scripts/rabbitmq-server.ocf

```diff
@@ -144,6 +147,14 @@ Location of the file in which the pid will be stored
 <content type="string" default="${OCF_RESKEY_pid_file_default}" />
 </parameter>
 
+<parameter name="limit_nofile" unique="0" required="0">
+<longdesc lang="en">
+Soft and hard limit for NOFILE
+</longdesc>
+<shortdesc lang="en">NOFILE limit</shortdesc>
+<content type="string" default="${OCF_RESKEY_limit_nofile}" />
```

Comment:
Should it be  `_default`  here to? Otherwise it's inconsistent between 2 scripts.